tests/NHSD.BuyingCatalogue.Ordering.Api.UnitTests/Builders/ServiceRecipientModelBuilder.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using NHSD.BuyingCatalogue.Ordering.Api.Models;
+using NHSD.BuyingCatalogue.Ordering.Api.UnitTests.Builders;
+using NHSD.BuyingCatalogue.Ordering.Domain;
+
+namespace NHSD.BuyingCatalogue.Ordering.Api.UnitTests.Controllers
+{
+    internal sealed class ServiceRecipientModelBuilder
+    {
+        private readonly ServiceRecipientModel _serviceRecipientModel;
+
+        private ServiceRecipientModelBuilder()
+        {
+            _serviceRecipientModel = new ServiceRecipientModel
```

Comment:
We agreed on Friday that the builders would use the slightly different style. So in this instance the `ServiceRecipientModel` object would be created in the `Build` method, not the constructor.